docs/testing.md

```diff
@@ -75,21 +75,21 @@ see [available settings](#available-settings-to-be-set-by-environment-variables)
 
 #### the manual way (e.g. to run from an existing ocis location)
 1. clone and build [ocis](https://github.com/owncloud/ocis)
-2. From inside the `phoenix` directory run `yarn run testing-app` to get the [testing-app](https://github.com/owncloud/testing), it's needed to have the skeleton folder for the tests
+2. From inside the `web` directory run `yarn run testing-app` to get the [testing-app](https://github.com/owncloud/testing), it's needed to have the skeleton folder for the tests
 3. Run redis server using docker
     ```sh
     yarn run redis-server
     ```
 
 4. Run the OCIS server with the necessary configurations
     ```sh
-    export PHOENIX_ASSET_PATH='<path-to-phoenix-clone>/dist'
-    export PHOENIX_WEB_CONFIG='<path-to-phoenix-clone>/dist/config.json'
+    export WEB_ASSET_PATH='<path-to-web-clone>/dist'
+    export WEB_WEB_CONFIG='<path-to-web-clone>/dist/config.json'
```

Comment:
Keep it? Or change to WEB_WEBUI_CONFIG
alternative WEB_CONFIG

Reply:
What about `WEB_UI_CONFIG`? `WEB_CONFIG` is supposed to be reserved for the viper config file.

Reply:
Yep, `WEB_UI_*` is what we went with in the ocis-web. Just forgot to adjust it here. Thanks for spotting!